src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -30,27 +34,13 @@ class TierCalculationService(
   fun calculateTierForCrn(crn: String): TierCalculationEntity {
     log.debug("Calculating tier for $crn using 'New' calculation")
 
-    val isFemale = communityApiDataService.isFemaleOffender(crn)
-
-    val protectScores = ProtectScores(
-      crn = crn,
-      mappaLevel = communityApiDataService.getMappa(crn),
-      rsrScore = communityApiDataService.getRSR(crn),
-      roshScore = communityApiDataService.getRosh(crn),
-      complexityFactors = communityApiDataService.getComplexityFactors(crn),
-      assessmentComplexityFactors = if (isFemale) assessmentApiDataService.getAssessmentComplexityAnswers(crn) else emptyMap()
-    )
-
-    val changeScores = ChangeScores(
-      crn = crn,
-      ogrsScore = communityApiDataService.getOGRS(crn),
-      need = assessmentApiDataService.getAssessmentNeeds(crn)
-    )
+    val protectLevel = calculateProtectLevel(crn)
+    val changeLevel = calculateChangeLevel(crn)
```

Comment:
i've left this as a seperate val because we're just going to change it with the mandate for change work